### PR TITLE
[TAS-541] 🚸 Allow copy wallet in feed page

### PR DESCRIPTION
**Pull request by nwingt**
image

### Mobile
Screenshot 2023-11-03 at 02 43 36

### CODE REVIEW

File: src/components/UserInfoCard.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<template>
+  <CardV2 class="flex flex-col items-center w-full gap-[1rem]">
+    <Identity
+      :class="{ 'cursor-pointer hover:scale-105 transition-transform': $listeners['click-avatar'] }"
+      :avatar-url="avatarSrc"
```
Comment (williamchong) [inline src/components/UserInfoCard.vue:5]: should also set displayname here?

Comment (williamchong) [inline src/components/UserInfoCard.vue:5]: should also set displayname here?

Comment (nwingt) [reply src/components/UserInfoCard.vue:5]: You mean `alt`?

Comment (williamchong) [reply src/components/UserInfoCard.vue:5]: Yes it is for `alt` but I think the prop is called `displayName`

Comment (nwingt) [reply src/components/UserInfoCard.vue]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**